docs/understand/sizing-guide.md

```diff
@@ -49,30 +49,36 @@ we recommend allocating at least 10 GiB - since it's a caching layer over a rela
 see [Important metrics](#important-metrics) below to understand how to size this: it should be big enough to hold all commit metadata for actively referenced commits.
 
 
-### PostgreSQL database
+### Metadata Store
```

Comment:
Will use 'Database' or 'Store' and not metadata store as it very easy to mix in our eco system what is metadata.